src/main/resources/application.properties

```diff
@@ -1 +1,24 @@
+server.port= 8085
+## Stripe API
+stripe.key.publishable=pk_test_ZPLk0o0siEhURnWP3O31PAxl003mdYeZGh
+stripe.key.secret=sk_test_68AFqEufTbTSfjsBBmPC8hdd00xKfiq7ze
 
+## PostgreSQL
+spring.datasource.platform=postgresql
+spring.datasource.url=jdbc:postgresql://localhost:5432/payment
+spring.datasource.username=root
+spring.datasource.password=666
+
+
+#drop n create table again, good for testing, comment this in production
+
+#spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=none
+spring.jpa.generate-ddl=true
```

Comment:
generate-ddl is a subset of hibernate.ddl-auto. You shouldn't specify both. generate-ddl equal to true means `hibernate.ddl-auto=update`. But you set it to none, so configs clash. As you use Liquibase, it'd be better to remove `generate-ddl` and use `hibernate.ddl-auto=validate` to make sure your entities compatible with tables you set up with Liquibase